src/features/Chat/components/Menu/Item.tsx

```diff
@@ -42,6 +42,8 @@ export const MenuItem = ({ buddy }: Props) => {
     return 'redDark';
   };
 
+  const showUnseenDot = activeFolder === 'ok' && hasUnread;
```

Comment:
I dont really care for these `activeFolder === 'ok'` checks in the code.
Maybe rather we should have a selector or a fn-to give a nice booleani instead. 
`isDefaultFolder(activeFolder)` or `selectIsDefaultFolder` etc.

Reply:
btw, the naming here would be better as `isUnseenDotVisible`, because `show` is imperative and sounds like a function

Reply:
Refactored!